backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -137,12 +137,12 @@ public Role getRole() {
    * Endpoint to update the information of a staff member.
    *
    * @param updatedStaff Updated staff information.
-   * @return The updated {@link Staff} member.
+   * @return The updated {@link Staff} list.
```

Comment:
I think this way is fine for now as we will not have many employees, but returning the updated staff member and replacing the one in the state with the updated version from the server is definitely the way to do it if we were dealing with more.